src/client.rs

```diff
@@ -521,8 +516,11 @@ where
             // SET SHARDING KEY TO 'bigint';
 
             let mut message = tokio::select! {
```

Comment:
In this case we would exit an admin client connection when shutdown is received. We want to allow those to stay healthy as long as possible (just as pgbouncer does)